src/services/cssCompletion.ts

```diff
@@ -54,6 +59,9 @@ export class CSSCompletion {
 				let node = this.nodePath[i];
 				if (node instanceof nodes.Property) {
 					this.getCompletionsForDeclarationProperty(node.getParent() as nodes.Declaration, result);
+					if (this.emmetCallback) {
+						this.emmetCallback();
```

Comment:
I initially had this callback add proposals to the current result right here in the language service. Doing so would mean the cache of css completions we want to maintain in the server side would contain the emmet completions as well. This would result in looping through the whole list and removing the older emmet completions each time the emmet results had to be updated.

If the actual adding of emmet proposals to the final output is done in the server side instead, then the cache to be maintained can have just the non emmet completions, thus avoiding looping through of the list each time the emmet results have to be updated.

Thoughts?